tests/integration/test_database/test_models/__init__.py

```diff
@@ -0,0 +1,77 @@
+from abc import ABCMeta
+
+from tests.integration import IntegrationTestCase
+from uuid import uuid4
+from sqlalchemy.orm import Session
+from topchef.database.models import Service
+
+
+class IntegrationTestCaseWithService(IntegrationTestCase, metaclass=ABCMeta):
+    """
+    Base class for integration tests, with a service already built
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        IntegrationTestCase.setUpClass()
+        cls.service_id = uuid4()
+        cls.service_name = 'Service'
+        cls.service_description = 'description'
+
+        cls.job_registration_schema = {
+            'type': 'object',
```

Comment:
We should make a JSON-schema generator, along with a JSON generator, that would work with ``hypothesis``. This would also be a cool thing to contribute upstream to the project.